attic/readme.md

```diff
@@ -10,3 +10,6 @@ Code, documentation, experiments we want to retain for reference, but that we're
     * note: somewhat tailored to use TI estimates, constant volume simulations, GRPC, ...
 * `scripts/`
   * `bootstrap_am1.py` -- script for approximating AM1BCC with atom types rather than bond types
+  * `rhfe_single.py` -- script for running relative hydration free energy with single topology
```

Comment:
Just noticed that these are stilled referenced here: https://github.com/proteneer/timemachine/blob/master/examples/readme.md

Reply:
Moved the descriptions to the attic readme